src/main/java/net/sf/mzmine/modules/visualization/spectra/simplespectra/spectraidentification/spectraldatabase/SpectraIdentificationResultsWindow.java

```diff
@@ -18,17 +18,16 @@
 
 package net.sf.mzmine.modules.visualization.spectra.simplespectra.spectraidentification.spectraldatabase;
```

Comment:
Can you briefly describe what you have changed here? Is the sorting now done after every new match?

Reply:
Yes. I have changed it to a list of SpectraDBIdentity 
after adding a match or multiple matches the whole thing is automatically sorted and the gridlayout is renewed.

Reply:
Okay, why not, it is a little bit more user friendly. I sorted all matches at the end. The sorting and redrawing was done in a ms, but if this is also fast, than it is better.

Reply:
Yeah it is very fast and as matches are done in batches it is no problem

Reply:
great job